USSF-ORBIT/ussf-portal-client
# feat: Display CMS content from hosted Keystone

# SC-147



## Proposed changes


- Replaces embedded CMS content with hosted content





## Setup


Note: this PR is blocked by [chore: Test fetching content from Keystone](https://github.com/USSF-ORBIT/ussf-portal-client/pull/601)

To test:
- Run `yarn services:up` in your local directory
- Run ussf-portal-cms locally with `yarn dev`
- Pull down this branch and run locally with `yarn dev`
- Verify that you can see content in My Space, click the 'Add link' button and verify that a list of Bookmarks to choose from appears
- Navigate to the Sites & Applications page, and verify that you can see Collections
- Logged in as cmsadmin, navigate to the CMS and update either a Collection or Bookmark, and verify that those changes are reflected in the portal client upon refreshing the page.
---

## Code review steps

### As the original developer, I have

  - ... 
  - [ ] Including cypress-axe checks when UI changes
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
- For any new migrations/schema changes:

### As code reviewer(s), I have

- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
  - [ ] On a mobile device in Firefox
  - [ ] On a mobile device in Chrome
  - [ ] On a mobile device in Safari
- [ ] Added any anomalous behavior to this PR
